projects/packages/forms/src/blocks/contact-form/editor.scss

```diff
@@ -240,12 +240,16 @@
 	width: 100%;
 }
 
+.jetpack-contact-form__thankyou-redirect-url {
+	min-width: auto;
```

Comment:
This overrides core styles: https://github.com/WordPress/gutenberg/blob/trunk/packages/block-editor/src/components/url-input/style.scss#L13. This should be handled in GB because it seems we cannot use `URLInput` in Inspector controls properly.

Reply:
Worth opening a github issue describing what is needed?

Reply:
Done: https://github.com/WordPress/gutenberg/issues/68659.